src/BccCode.Linq/ApiClient/ResultList.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System.Collections;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+namespace BccCode.ApiClient;
+
+[DataContract(Name = "ResultList")]
+public abstract class ResultList
+{
+    protected ResultList(Metadata? meta = null)
+    {
+    }
+
+    /// <summary>
+    /// Gets or Sets Meta
+    /// </summary>
+    [DataMember(Name = "meta", EmitDefaultValue = false)]
+    public Metadata? Meta { get; set; }
+
+    [JsonIgnore]
+    public IEnumerable<object>? Data { get; set; }
```

Comment:
This should probably be moved to the derived class, and changed to IEnumerable<T>

Reply:
Yes. Actually, I am not sure if we need that `ResultList` class at all. `ResultList<T>` is enough I think. and it does not seem to be in use in the Client SDK as far as I can see.

Reply:
`ResultList` is now dropped